### ISSUE TITLE
Build the first page of registering an early career teacher for schools

### ISSUE THREAD

**Issue by claire-hughez** [DFE-Digital/register-early-career-teachers#430]
## Why?
We're fairly confident with our design for the first part of registering an early career teacher.

We need to tell school users what information they'll need to register an ECT.

We'll also need to ask for the ECT's TRN and DOB, so we can check they are actually a teacher and they're eligible against the DQT API.

## What?

**IN SCOPE**
- Build the first page to tell school users what they'll need to register an early career teacher

**OUT OF SCOPE** (as per dev refinement session 8 Oct 2024)
- Build the second page where they give TRN and date of birth for the early career teacher
- Build the functionality to check the TRS API for this information and send us back the name

![Image](https://github.com/user-attachments/assets/02409e7f-3512-49af-8a34-d54c762272f0)

![Image](https://github.com/user-attachments/assets/936875b8-087a-4f1e-a484-ed4fba8bacda)

## Out of scope
- Any other pages that are part of adding an early career teacher
- Any sort of homepage users see when they've signed in
- Anything for adding an ECT that involves asking for appropriate body, lead provider or delivery partner details
- Validation errors, for now!
- Actually working out the eligibility rules for the DQT check

## Helpful links or other information
[Link to the prototype](https://ecf2-prototy-sprint-8-s-znawg0.herokuapp.com/v2/school/what-youll-need)

We should be able to use work that has been done for the AB portal replacement to help with this.

**Comment by peteryates** [comment]
I think this depends on #383 so let's bring that into this sprint too.
